OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -19,9 +19,13 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 	{
 		[Desc("HitPoints")]
 		public readonly int HP = 0;
+
 		[Desc("Trigger interfaces such as AnnounceOnKill?")]
 		public readonly bool NotifyAppliedDamage = true;
 
+		[Desc("Distance above CenterPosition for targeting by weapons.")]
```

Comment:
Directional armor doesn't at all relate to the projectile's initial target position.
The "hassle" is literally changing this to a `WVec` then changing the referencing code to use this directly instead of creating a `WVec` from components.

But ok, this can be done later when someone requests it.